pkg/detectors/browserstack/browserstack_test.go

```diff
@@ -101,13 +104,37 @@ func TestBrowserStack_FromChunk(t *testing.T) {
 				data:   []byte(fmt.Sprintf("You can find a BROWSERSTACK_ACCESS_KEY %s within BROWSERSTACK_USERNAME %s", secret, secretUser)),
 				verify: true,
 			},
-			want: []detectors.Result{
-				{
+			want: func() []detectors.Result {
+				r := detectors.Result{
 					DetectorType: detectorspb.DetectorType_BrowserStack,
 					Verified:     false,
 					RawV2:        []byte(fmt.Sprintf("%s%s", secret, secretUser)),
-				},
+				}
+				r.SetVerificationError(fmt.Errorf("unexpected HTTP response status 404"), secret)
+				results := []detectors.Result{r}
+				return results
+			}(),
+			wantErr:             false,
+			wantVerificationErr: true,
```

Comment:
It looks like you can delete the flag in this test case because you're setting the verification error explicitly in the test case - can you delete the flag for all of them?

Reply:
yep. nice catch